components/admin_console/billing/plan_details/index.tsx

```diff
@@ -22,8 +22,7 @@ type PlanDetailsProps = {
     isFreeTrial: boolean;
     subscriptionPlan: string | undefined;
 }
-/* eslint-disable react/prop-types */
-const PlanDetails: React.FC<PlanDetailsProps> = ({isFreeTrial, subscriptionPlan}) => {
+const PlanDetails = ({isFreeTrial, subscriptionPlan}: PlanDetailsProps) => {
```

Comment:
this is not related to the PR, but is part of the effort to clean up components that I see using React.FC unnecessarily.